diff --git a/v3/src/utf8_codecvt_facet.cpp b/v3/src/utf8_codecvt_facet.cpp
index 1849a1a..806a300 100644
--- a/v3/src/utf8_codecvt_facet.cpp
+++ b/v3/src/utf8_codecvt_facet.cpp
@@ -16,7 +16,7 @@
 #define BOOST_UTF8_END_NAMESPACE }}}
 #define BOOST_UTF8_DECL BOOST_FILESYSTEM_DECL
 
-#include "libs/detail/utf8_codecvt_facet.cpp"
+#include "boost/detail/utf8_codecvt_facet_impl.hpp"
 
 #undef BOOST_UTF8_BEGIN_NAMESPACE
 #undef BOOST_UTF8_END_NAMESPACE
