diff --git a/test/test_xml_parser_rapidxml.cpp b/test/test_xml_parser_rapidxml.cpp
index 5a0e3d4..1843fad 100644
--- a/test/test_xml_parser_rapidxml.cpp
+++ b/test/test_xml_parser_rapidxml.cpp
@@ -15,7 +15,7 @@
 #define BOOST_UTF8_END_NAMESPACE }}
 #define BOOST_UTF8_DECL
 #include <boost/detail/utf8_codecvt_facet.hpp>
-#include "../../detail/utf8_codecvt_facet.cpp"
+#include "boost/detail/utf8_codecvt_facet_impl.hpp"
 
 int test_main(int argc, char *argv[])
 {
